source-code/core2/src/fullExample.ts

```diff
@@ -93,12 +89,13 @@ inlang.messages.query.update({ where: { id: "myMessageId", languageTag: "en" },
 inlang.messages.query.delete({ where: { id: "myMessageId", languageTag: "en" } })
 
 // --- CONFIG ACCESS ---
-//   - config is static. adjusting the config requires a re-initialization of the app.
-//     IMHO (@samuelstroschein) that okay. 
 
 inlang.config.sourceLanguageTag
 inlang.config.languageTags
 
+// adding a language tag via an app
+inlang.config.languageTags = [...inlang.config.languageTags, "de-AT"]
```

Comment:
config is mutable then? 😅

Reply:
I mean everything is mutable in inlang anyways. But you want read/write separation? 

We could have an API like 

```ts
// get a prop
inlang.config.get()

// set a prop
inlang.config.set()

```

But, that really depends on #1122